infra/docker/README.md

```diff
@@ -1,7 +1,10 @@
 # Using Docker
-The ec2 instance must have the training data available at ~/data.
+The ec2 instance must have the training data available at ~/data. You also need to have Nvidia driver, docker and nvidia-docker installed on the AMI.
```

Comment:
Should we add a link for the Nvidia driver installation?
https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html

Reply:
I think that will be too detailed. User can search by themselves. But you can feel free to add it if you want.